strawberry/extensions/tracing/opentelemetry.py

```diff
@@ -120,15 +108,17 @@ async def resolve(self, _next, root, info, *args, **kwargs):
 
             return result
 
-        with _use_span(self._span_holder[RequestStage.REQUEST], self._tracer):
-            with self._tracer.start_span(info.field_name, kind=SpanKind.SERVER) as span:
-                self.add_tags(span, info, kwargs)
-                result = _next(root, info, *args, **kwargs)
+        with self._tracer.start_as_current_span(
+            f"GraphQL Handling: {info.field_name}",
```

Comment:
Maybe 'resolving' would be more specific than 'handling'? Not sure.

Reply:
I like it! Change coming up.